CMakeLists.txt

```diff
@@ -676,6 +678,7 @@ if(${use_installed_dependencies})
         FILES
             "configs/${PROJECT_NAME}Config.cmake"
             "configs/${PROJECT_NAME}Functions.cmake"
+           "configs/azure_iot_build_rules.cmake"
```

Comment:
>"configs/azure_iot_build_rules.cmake" [](start = 11, length = 37)

Nit - start of line does not align with others in same block.

Reply:
to be fixed.

Reply:
Done.